UrCan/HW3/PokemonBase.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PokemonBase {
```

Comment:
Ik mis de Attack int die wel in Niels' UML Diagram staat onder PokemonBase

Reply:
Vergeten. Goede opmerking